openforcefield/topology/molecule.py

```diff
@@ -3176,6 +3248,164 @@ def from_openeye(oemol, allow_undefined_stereo=False):
         return toolkit.from_openeye(
             oemol, allow_undefined_stereo=allow_undefined_stereo)
 
+    def to_qcshema(self):
+        """
+
+        :return:
+        """
+        raise NotImplementedError
+
+    @classmethod
+    def from_qcarchive(cls, qca_mol, smiles=None, allow_undefined_stereo=False):
```

Comment:
Should this be `from_qcschema` to match `to_qcschema`?

Reply:
Not sure about the naming here `from_qcshema` would suggest that the molecule is being made from the schema info like geometry, connectivity and elements, but this function only works at the entry-level of the archive which has the cmiles identifiers where its safe to remake the molecule. Maybe something like `from_qcentry`  ? Then we could also have a `from_qcschema` method which could try and construct the molecule using the schema info which is in every record with a warning?